appliances/backup_and_restore.md

```diff
@@ -35,17 +35,22 @@ Before adding the cron job it would be best practice to mount an external storag
 # Restore
 Restoring a backup of the appliance will be done using SSH or the console.
 
-1. Download a backup TGZ and place it in `/`
-2. On the command line:
+1. Set the hostname on the new appliance.  On the command line:
+   ```bash
+     hostnamectl hostname your-new-hostname.example.com
+     echo "your.ip.address your-new-hostname.example.com your-new-hostname" >> /etc/hosts
+   ```
+2. Download a backup TGZ and place it in `/`
+3. On the command line:
    ```bash
      cd /
-     tar -zxf /backup
-     rm -f /backup # if desired to save space
+     tar -zxf /backup.tgz
+     rm -f /backup.tgz # if desired to save space
    ```
-3. Launch `appliance_console`
-   1. If this is a brand new appliance and the database has not yet been initialized, select `Configure Application`. If this is an existing database appliance, this step can be skipped.
+4. Launch `appliance_console`
+   1. Since the database has not yet been initialized, select `Configure Application`.
       1. When prompted to configure the database, select `Create Internal Database`
-      2. When prompted to configure messaging, select `Make No messaging changes`
+      2. When prompted to configure messaging, either connect to an external messaging system or configure this appliance as a messaging server
```

Comment:
Should we say something such as:

If you are planning on restoring a database to this system, please verify the encryption key, v2_key, for use with that database was properly extracted from the backup.tgz into the certs directory.  Ideally, we could add a prompt in the console where it displays the "generate new key" screen to also prompt that an existing v2_key was found in the else case.  I'm not sure how we streamline this other than to prompt for the backup.tgz and we extract it and look for the v2_key and warn if it's missing.